tools/identity-resolution/Helpers/GitHubToAADConverter.cs

```diff
@@ -73,5 +74,26 @@ public async Task<string> GetUserPrincipalNameFromGithubAsync(string githubUserN
 
             return null;
         }
+
+        public async Task<UserLink[]> GetPeopleLinksAsync()
+        {
+            try
+            {
+                var responseJsonString = await client.GetStringAsync($"https://repos.opensource.microsoft.com/api/people/links");
```

Comment:
Does this call get logged anywhere? Especially if someone is running this locally it might help to let them know what they are waiting on?